integr8ly/application-monitoring-operator
# Prepare release 1.7.0 - Add token to Blackbox Exporter ConfigMap

# Description
Version bump to 1.7.0 that fixes the Endpoint Detailed dashboard that would incorrectly show the grafana UI endpoint as down. This bugfix adds a ServiceAccount with namespace:get permissions and adds the ServiceAccount's token to the Blackbox Exporter ConfigMap so the Blackbox probe requests can reach the grafana endpoint that is behind a proxy.

JIRA: https://issues.redhat.com/browse/MGDAPI-2282

## Verification
1. Create a new OSD cluster via [OpenShift](https://qaprodauth.cloud.redhat.com/openshift/) and target it once it has finished provisioning.
2. Pull [this PR](https://github.com/integr8ly/integreatly-operator/pull/2052) from the integreatly-operator repo and change [this line](https://github.com/integr8ly/integreatly-operator/blob/4fe1c29837037879fb806512cd9870ae4b6d2ac8/manifests/integreatly-monitoring/1.7.0/application-monitoring-operator.v1.7.0.clusterserviceversion.yaml#L238) to point to the image stored in my Quay [repository](https://quay.io/repository/ckyrillo/application-monitoring-operator):
`image: quay.io/ckyrillo/application-monitoring-operator:v1.7.0`
3. Install RHOAM from the command line to the remote cluster using the PR from Step 2.
4. Once the installation is complete, verify that the "grafana-ui UP/DOWN Status" panel in the Endpoint Dashboard from the redhat-rhoam-middleware-monitoring-operator grafana route correctly displays that the grafana service is UP.

Screenshot of the dashboard without the bugfix:
![MGDAPI-2282-bug](https://user-images.githubusercontent.com/80267718/129981866-50ce33ff-d87f-4593-8875-edb5ac8e88da.png)

Screenshot of the dashboard with the bugfix:
![MGDAPI-2282-bugfix](https://user-images.githubusercontent.com/80267718/129980186-8493c3f8-b11f-41d7-9abb-3a42828fe232.png)


## Type of change

## Checklist
